backend/app/services/config_service.py

```
@@ -0,0 +1,135 @@
+import json
+import sqlite3
+from pathlib import Path
+from typing import Optional
+
+from app.models.config import AppConfiguration, SiteConfiguration, UserPreferences
+
+
+class Configuration:
+    def __init__(self, db_path: str = "cv_autofill.db"):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)